### ISSUE TITLE
Why use rgb refine degenerates the results?

### ISSUE THREAD

**Issue by Richardych** [DanaCohen95/Set-the-Scene#4]
Have you tried to perform rgb refinement based on the pre-trained latent mode, like latent-nerf?
However, I tried it and the results were poor.

**Comment by DanaCohen95** [comment]
Hi!
I did try to perform rgb refinement, but similar to you, I didn't get better results. The results I got were over-saturated.

**Comment by Richardych** [comment]
> Hi! I did try to perform rgb refinement, but similar to you, I didn't get better results. The results I got were over-saturated.

@DanaCohen95 yes, I got the same over-saturated results. Do u have any ideas about this issue?
